### PR TITLE
feat: progress bar in challenge header

### PR DESCRIPTION
**Pull request by christian14b**
Updating campaign progress bar to handle amounts instead of people and adding an Icon at the end.

image

### CODE REVIEW

**Comment by roger-in-kiva** [comment]
@christian14b Missing shadow for the icon placeholder.

File: src/components/Kv/KvProgressCampaign.vue [hunk 1]
```diff
@@ -2,46 +2,51 @@
...
-			:bg-variant="bgVariant"
-		/>
-		<div v-if="!minimalStats">
-			<p>
-				{{ daysLeft }} days left
-			</p>
+		<div class="tw-relative tw-mt-1">
+			<kv-progress-bar
+				class="tw-my-0.5"
+				aria-label="Percent the loan has funded"
```
Comment (dyersituations) [inline src/components/Kv/KvProgressCampaign.vue:14]: `Percent the campaign has funded`

File: src/components/Kv/KvProgressCampaign.vue [hunk 2]
```diff
@@ -2,46 +2,51 @@
...
 export default {
 	name: 'KvProgressCampaign',
 	components: {
-		KvProgressBar
+		KvProgressBar,
+		HeartOutFromBox
 	},
 	props: {
-		fundedBorrowers: {
+		fundedAmount: {
```
Comment (dyersituations) [inline src/components/Kv/KvProgressCampaign.vue:37]: With these prop changes, make sure to remove it from the existing usage, `IwdProgressCampaign`. We still might be user testing the landing page, so we need to make sure it still loads, but the progress bar won't be shown without an active campaign, so we can remove it from that component.

File: src/components/Lend/LoanSearch/ChallengeHeader.vue [hunk 3]
```diff
@@ -1,17 +1,42 @@
...
+		fundedAmount() {
+			// TODO: Change this when we have the correct data name
+			return this.challengeData?.amountGoal ?? 0;
+		},
+		totalAmount() {
+			return this.challengeData?.participation?.amountLent ?? 0;
+		},
+		daysLeft() {
+			return intervalToDuration({
+				start: this.challengeData?.endDate ?? new Date(),
```
Comment (dyersituations) [inline src/components/Lend/LoanSearch/ChallengeHeader.vue:36]: `startDate`?

File: src/components/Kv/KvProgressCampaign.vue (hunk 2 above)
Comment (dyersituations) [reply src/components/Kv/KvProgressCampaign.vue:37]: Let's keep that component for now, just comment out the `kv-progress-campaign`. We still don't know the future of that IWD page.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**